io.designer.GravitDesigner.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.designer.GravitDesigner</id>
+  <launchable type="desktop-id">io.designer.GravitDesigner.desktop</launchable>
+  <name>GravitDesigner</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Corel Corporation</developer_name>
+  <summary>Vector design app</summary>
+  <description>
+    <p>
+      Gravit Designer is a full-featured, free vector design app right at your fingertips.
```

Comment:
Can you add the standard language about this being an extra-data wrapper here? (like this stuff https://flathub.org/apps/details/com.syntevo.SmartGit )